logverification/validation.go

```diff
@@ -17,20 +21,20 @@ var (
 
 // Validate performs basic validation on the VerifiableEvent, ensuring that critical fields
```

Comment:
Does this comment need updating?

Reply:
yes! good spot!

there is alot of event -> app entry baggage, so if you spot anymore please shout :)

Reply:
done!